vm/src/vm/errors/vm_exception.rs

```diff
@@ -1030,7 +1030,7 @@ cairo_programs/bad_programs/uint512_unsigned_div_rem_div_is_zero.cairo:15:2: (pc
     fn run_bad_uint256_sub_check_error_displayed() {
         #[cfg(feature = "std")]
         let expected_error_string = r#"cairo_programs/bad_programs/uint256_sub_b_gt_256.cairo:17:1: Error at pc=0:17:
-Got an exception while executing a hint: Inconsistent memory assignment at address Relocatable { segment_index: 1, offset: 6 }. Int(1) != Int(41367660292349381832802403122744918015)
+Got an exception while executing a hint: Inconsistent memory assignment at address Relocatable { segment_index: 1, offset: 6 }. Int(Felt(FieldElement { value: UnsignedInteger { limbs: [576460752303422960, 18446744073709551615, 18446744073709551615, 18446744073709551585] } })) != Int(Felt(FieldElement { value: UnsignedInteger { limbs: [461241316192151699, 18446744073709551612, 2025524839466148960, 6944660869770031956] } }))
```

Comment:
Error messages should not change. There are users relying on them. Besides, the message showing a number is much more user friendly.